docs: Update README with mise installation command

airtonix opened the pull request:
Added installation command for using 

```
mise use --pin -g ubi:sst/opencode
```

jayair commented:
Can you remove the other lines changes from this commit? Not sure if you made them or it's just the diff.

airtonix commented:
> Can you remove the other lines changes from this commit? Not sure if you made them or it's just the diff.

Yep no worries, not sure what those other changes are tbh. All I changed was the one line. Maybe i have the wrong base? 

Also noticed that there's three places where install method is mentioned: 

- the repo README.md (where I originally made a change)
- the website/content/docs/index.md
- the website/components/Lander.astro

I've added mise there. 

Personally I think you shouldn't be promoting `curl | bash` method, instead promote [ubi](https://github.com/houseabsolute/ubi) or [mise](https://mise.jdx.dev/) over all the others.

airtonix committed:
docs: add mise to install methods
